File: backend/src/main/java/com/codeloom/backend/dao/testcase/TestCaseResultRepository.java
package com.codeloom.backend.dao.testcase;

import com.codeloom.backend.model.TestCaseResult;
import java.util.UUID;
import org.springframework.data.repository.CrudRepository;

public interface TestCaseResultRepository extends CrudRepository<TestCaseResult, UUID> {
  void deleteBySubmissionId(UUID id);
}

File: backend/src/main/kotlin/com/codeloom/backend/dao/testcase/TestCaseResultRepository.kt
package com.codeloom.backend.dao.testcase

import com.codeloom.backend.model.TestCaseResult
import org.springframework.data.repository.CrudRepository
import java.util.UUID

interface TestCaseResultRepository : CrudRepository<TestCaseResult, UUID> {
    fun deleteBySubmissionId(submissionId: UUID)
}
